File: backend/routes/twilio_routes.go
package routes

import (
	"eunity.com/backend-main/controllers"
	"github.com/gin-gonic/gin"
)

var Twilio_controllers = controllers.Twilio_controllers{}

// TwilioRoutes sets up the routing for Twilio related functionality
func Twilio_routes(r *gin.RouterGroup) {
	r.POST("/send-sms", Twilio_controllers.Send_ver_sms)

	r.POST("/verify-phone", Twilio_controllers.Verify_phone)
}

// sendSMSHandler handles sending SMS messages

File: python/routes/twilio_routes.py
from fastapi import APIRouter
from controllers import twilio_controllers as twilio_controller


# Create an instance of the TwilioController

# Create an instance of the APIRouter with a prefix and tags
router = APIRouter()


# Define the route for sending a verification SMS
@router.post("/send-sms")
async def send_ver_sms():
    return twilio_controller.send_ver_sms()


# Define the route for verifying a phone number
@router.post("/verify-phone")
async def verify_phone():
    return twilio_controller.verify_phone()